ibm-cloud-pak-for-data/README.md

```diff
@@ -225,7 +200,22 @@ az group deployment create --resource-group <resource_group> --template-file azu
 ![Alt text](images/Output.png?raw=true "output")
 <br/><br/>
 
-Use admin / password to log in to CPD console.
+Use the default credentials for Cloud Pak for Data admin / password to log in to CPD console. Ensure to change the password after your first login.
+
+### Method 2: Using Azure CLI
```

Comment:
remove this section it's the same for all samples in the repo

Reply:
Done